tox.ini

```diff
@@ -0,0 +1,28 @@
+[tox]
+envlist = py{27,35}
+skipsdist = true
+
+[pep8]
+max-line-length = 120
+
+[pydocstyle]
+ignore = D200,D203,D212
+
+[pytest]
+testpaths = .
+norecursedirs = .* requirements
+
+[testenv]
+deps =
+    -rrequirements.txt
+commands =
+    python test_pipeline.py
+
+[testenv:quality]
+deps =
+    -rrequirements.txt
+commands =
+    pylint composer.py message.py pipeline_manager.py test_pipeline.py
```

Comment:
These files should be moved into a common notifications package, but I don't want to make that change on this branch, because it will cause merge problems with the existing work.